pkg/controller/user-cluster-controller-manager/resources/resources/envoy-agent/daemonset.go

```diff
@@ -147,14 +159,14 @@ func getInitContainers(ip net.IP, versions kubermatic.Versions, registryWithOver
 				},
 			},
 		},
-	}
+	}, nil
 }
 
-func getContainers(versions kubermatic.Versions, registryWithOverwrite registry.WithOverwriteFunc) []corev1.Container {
+func getContainers(versions kubermatic.Versions, imageRewriter registry.ImageRewriter) ([]corev1.Container, error) {
 	return []corev1.Container{
 		{
 			Name:            resources.EnvoyAgentDaemonSetName,
-			Image:           fmt.Sprintf("%s/%s:%s", registryWithOverwrite(resources.RegistryDocker), envoyImageName, versions.Envoy),
+			Image:           registry.Must(imageRewriter(fmt.Sprintf("%s:%s", envoyImageName, versions.Kubermatic))),
```

Comment:
Why is this changed from `versions.Envoy` to `versions.Kubermatic`?

Reply:
Ouh that was a copy-paste mistake.